server/Makefile-flashinfer

```diff
@@ -0,0 +1,2 @@
+install-flashinfer:
+	pip install flashinfer==0.1.5 -i https://flashinfer.ai/whl/cu124/torch2.4
```

Comment:
will this error on lower cuda versions? CI seems to be on 12.2 but also I'm not sure `make install-flashinfer` is included in the build commands. 

**tldr, just a bit confused about the flashinfer that will be used

Reply:
CI ? the dockerfile also is upgraded to 12.4 actually (was done before to set the CI green again)